### ISSUE TITLE
Keras-UFF model inference output is incorrect

### ISSUE THREAD

**Issue by Wheest** [NVIDIA/TensorRT#932]
Making a new issue, related to #375, since that one has been closed, and my issue does not involve ONNX.

In summary, exporting a model from Keras via UFF gives the incorrect output.

I have a [simple reproducible example of this](https://github.com/Wheest/keras_tensorrt_inference), where I don't even use ONNX, I just use the underlying TensorFlow model to import via UFF and thus into TensorRT.   I am testing on an AGX Xaviar.

My example is MobileNetV2, and the predictions are incorrect.  It's an ImageNet model, so the output is an array of 1000, so no possible reshaping issues.  The output is the expected size.

The inference code is just adapted from the official example `/usr/src/tensorrt/samples/python/introductory_parser_samples/uff_resnet50.py`, and I've verified that the label lookup table is correct.

The README in the example should have everything you need to understand how to run it.

My example is available on [this GitHub repo](https://github.com/Wheest/keras_tensorrt_inference).

**Comment by Wheest** [comment]
My issue was related different preprocessing steps between the Keras and TensorRT model.  See the updated MWE for the solution.
